### PR TITLE
storage_pdisk_state, storage_vdisk_state, and others

### PR DESCRIPTION
**Pull request by hbjydev**
Hi guys,

I've implemented some more fields pulled from omreport, we needed them for our monitoring. If you don't want them that's fine, but they're here if you do. :)

Not marking as complete until more fields are pushed and tested.

### CODE REVIEW

**Comment by hbjydev** [comment]
Re-pushed all commits with correct GPG signature.

File: pkg/omreport/util.go [hunk 1]
```diff
@@ -75,6 +75,82 @@ func severity(s string) string {
 	return "0"
 }
 
+func pdiskState(s string) string {
+	states := map[string]string{
```
Comment (galexrt) [inline pkg/omreport/util.go:79]: Would you be up to document these states in the documentation?
So others could more easily build their monitoring around these new metrics.

Comment (hbjydev) [reply pkg/omreport/util.go:79]: Ah, yeah, I knew there was something I forgot lol. I'll get on that shortly

Comment (galexrt) [reply pkg/omreport/util.go:79]: Thanks, really appreciate your PR!

**Comment by galexrt** [comment]
@hbjydev Would you be able to add the documentation for the new metrics states?

**Pull request closed without merging**